contentpacks/khanacademy.py

```diff
@@ -55,11 +58,11 @@ def new_merge(self, other):
 POEntry_class.merge = new_merge
 
 
-def retrieve_language_resources(version: str, sublangargs: dict, no_subtitles: bool) -> LangpackResources:
-    node_data = retrieve_kalite_data(lang=sublangargs["content_lang"], force=True)
+def retrieve_language_resources(version: str, sublangargs: dict, no_subtitles: bool, no_dubbed_videos) -> LangpackResources:
+    node_data = retrieve_kalite_data(lang=sublangargs["content_lang"], force=True ) if not no_dubbed_videos else {}
```

Comment:
I don't think we want to skip the entire topic tree retrieval when we give this flag. Since we keep the `en_nodes.json` around, maybe just return that instead of skipping the topic tree entirely?